os_site/src/blog.njk

```diff
@@ -0,0 +1,18 @@
+---
+layout: base.njk
+permalink: /posts/
+---
+
+{% from "macros/post-feed-item.njk" import postFeedItem %} 
+
+<main>
+    {% set postlist = collections.posts %}
+    <ul class="post-feed c-block-feed has-giant-btm-marg">
+        <div class="post-feed-title">
+            <h1> All Posts </h1>
+        </div>
```

Comment:
Would it be possible to move this div & h1 out of the `<ul>`? That way we only have a list items in there, which is a little more standard for that tag.

Reply:
Yes, originally did this so that the header would also have padding, but I see how that is not ideal under the list tag. Added a container for the feed to address this!

Reply:
Looks great! One not so obvious tip is if you ever need to throw spacing at an element, @AndrewGibson27 made [these `margin-bottom` helpers](https://texastribune.github.io/queso-ui/sections/utilities/has-size-btm-marg/) that come in handy for that. In this case you could do `<h1 class="has-l-btm-marg">All Posts</h1>`. Adding padding-bottom works just fine tho; accomplishes the same thing.

Reply:
Oh, these helpers are so great. Thank you!!